src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/AStar.java

```diff
@@ -0,0 +1,100 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+
+public class AStar {
+    private static final NodeBase nodeBase = NodeBase.getInstance();
+
+  public static double getDistanceFromCurrentNodeToEndOrStartNode(NodeArm point, NodeArm current){
```

Comment:
1. Function name refers to end/start nodes but it doesn't actually matter if it's a start or end node, right?
2. You could also change the argument names to be more readable, maybe `start` and `destination`?

Reply:
deleted the func